### ISSUE TITLE
Changelog is not sorted by date

### ISSUE THREAD

**Issue by myclevorname** [thunderbird/thunderbird-android#8577]
### Checklist

- [x] I have used the search function to see if someone else has already submitted the same bug report.
- [x] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.1

### Where did you get the app from?

F-Droid

### Android version

12

### Device model

Samsung Galaxy S10e

### Steps to reproduce

1. Go to Settings
2. Press About, then Version
3. Scroll down

### Expected behavior

Versions should be in order

### Actual behavior

Version 8.2b2 and 8.2b3 are in the wrong order

![Image](https://github.com/user-attachments/assets/f1f125dc-e986-46ac-af49-d47e1635da9b)

Bonus issue too small to be its own issue: The word "aka" in the Github repo description is misspelled as "fka"

**Comment by cketti** [comment]
@kewisch @coreycb The changelog is sorted by version code. The easiest solution I can see is to not include the changelog entries from `beta` in `release`.

**Comment by kewisch** [comment]
Agree, I think the fact that betas were included was a result of merges and not a clear idea of what to do with the changelog. Maybe we should generate the full changelog xml file from the (various) yaml files in the `android_beta` or `android_release` subdirectories each time, or we need to adjust/create a merge day script that will clean up the files accordingly.

**Comment by wmontwe** [comment]
We could use different files for the different build types. At the moment the file is located at `app-thunderbird/src/main/res/raw` but we could move it to:
- Daily: `app-thunderbird/src/daily/res/raw`
- Beta: `app-thunderbird/src/beta/res/raw`
- Release: `app-thunderbird/src/beta/res/raw`

So every variant will only include it's own changelog.

**Comment by kewisch** [comment]
Given recent discussions I think we should go ahead with per-channel release notes as @wmontwe suggested. Additions will be done in thunderbird-notes anyway, where they are also per-channel.

**Comment by wmontwe** [comment]
This is fixed by https://github.com/thunderbird/thunderbird-android/pull/8858
